_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw_utils_new.cpp

```diff
@@ -892,16 +892,16 @@ void MfxHwH264Encode::ModifyRefPicLists(
                                 std::swap(*l, *r);
                 }
             }
-            
+
             if (video.calcParam.numTemporalLayer == 0 && (task.m_type[0] & MFX_FRAMETYPE_P) && task.m_internalListCtrlRefModLTR)
             {
                 mfxU8 * begin = list0.Begin();
                 mfxU8 * end = list0.End();
                 mfxU8 * ltr = 0;
                 for (; begin != end; ++begin)
-                    if (dpb[*begin & 127].m_longterm) 
+                    if (dpb[*begin & 127].m_longterm)
```

Comment:
are you sure you want to fight with failing cherry-pick because of that?

Reply:
These are settings of my text editor.  And I don't want to change it.